src/base/rss/rss_autodownloadrule.cpp

```diff
@@ -393,6 +462,8 @@ AutoDownloadRule AutoDownloadRule::fromVariantHash(const QVariantHash &varHash)
     rule.setAddPaused(TriStateBool(varHash.value("add_paused").toInt() - 1));
     rule.setLastMatch(varHash.value("last_match").toDateTime());
     rule.setIgnoreDays(varHash.value("ignore_days").toInt());
+    rule.setUseSmartFilter(varHash.value("smart_filter", false).toBool());
```

Comment:
Speaking as someone who's been using a build of qBT with an older version of this PR merged in, I don't mind if I have to set up smart filters again or if it loses track of which episodes have downloaded when this feature is part of an official build. I pretty much figured that would happen at least once when I started testing the feature out.